verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -197,13 +248,16 @@ class KytheFactsExtractor {
 // Creates the signature for module names.
 std::string CreateModuleSignature(absl::string_view);
 
+// Creates the signature for package names.
+std::string CreatePackageSignature(absl::string_view);
+
 // Creates the signature for Class names.
 std::string CreateClassSignature(absl::string_view);
 
 // Creates the signature for module instantiations.
 std::string CreateVariableSignature(absl::string_view);
 
-// Creates the signature for funcitons or tasks.
+// Creates the signature for functions or tasks.
 std::string CreateFunctionOrTaskSignature(absl::string_view);
```

Comment:
Do these need to be in the public header (are they called outside of this library?), or could they be private-ized to the .cc implementation file?